cloudfoundry/cfapi/service_manager.go

```diff
@@ -505,6 +507,97 @@ func (sm *ServiceManager) ReadServiceInstance(serviceInstanceID string) (service
 	return serviceInstance, nil
 }
 
+// WaitServiceInstanceTo -
+func (sm *ServiceManager) WaitServiceInstanceTo(operationType string, serviceInstanceID string) (err error) {
+	sm.log.UI.Say("Waiting for service instance %s to finish starting ..", terminal.EntityNameColor(serviceInstanceID))
```

Comment:
Alternative wording suggestion: "Waiting for service instance %s to complete async provisionning"